libs/loggers/loggers.py

```diff
@@ -46,14 +54,42 @@ def update(self, objects_list, distances):
         """
 
         if time.time() - self.submited_time > self.time_interval:
+            # Get timeline which is used for as Timestamp
+            now = datetime.now()
+            current_time = now.strftime("%Y-%m-%d %H:%M:%S")
+            # Process objects
             objects = self.format_objects(objects_list)
-            self.logger.update(objects, distances, version=LOG_FORMAT_VERSION)
+            violating_objects = extract_violating_objects(distances, self.dist_threshold)
+            # Get unique objects that are in close contact
+            violating_objects_index_list = list(set(itertools.chain(*violating_objects)))
+            # Get the number of violating objects (people)
+            violating_objects_count = len(violating_objects)
+            # Get the number of detected objects (people)
+            detected_objects_count = len(objects_list)
+            # Get environment score
+            environment_score = mx_environment_scoring_consider_crowd(detected_objects_count, violating_objects_count)
+            self.logger.log_objects(
+                objects,
+                violating_objects,
+                violating_objects_index_list,
+                violating_objects_count,
+                detected_objects_count,
+                environment_score,
+                current_time,
+                version=LOG_FORMAT_VERSION
+            )
+            if self.web_hook_logger:
+                self.web_hook_logger.log_objects(
```

Comment:
As these two loggers have a log_objects function  with the same signature, I am tempted to do something like this:

```python
for logger in [logger1, logger2]:
    logger.log_objects(...)
```

That would be nice if you plan to support multiple loggers, otherwise might not be that helpful.

What do you think?

Reply:
I think that this is a good idea.

I changed the logger implementation structure following the pattern you suggested.